edge-hub/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -1,30 +1,17 @@
 ARG base_tag=2.1.6-runtime-bionic-arm32v7
-ARG num_procs=4
-
-FROM arm32v7/buildpack-deps:bionic as builder
-
-RUN apt-get update && \
-    apt-get install -y libsnappy-dev
-
-RUN git clone -b v5.4.6 --depth 1 https://github.com/facebook/rocksdb.git && \
-    cd rocksdb && \
-    CFLAGS="-static-libstdc++ -Wno-psabi -DSNAPPY" PORTABLE=1 make -j ${num_procs} shared_lib && \
-    strip librocksdb.so && \
-    mkdir -p ../publish && \
-    cp -vL librocksdb.so ../publish/
-
 FROM microsoft/dotnet:${base_tag}
 
 # Add an unprivileged user account for running Edge Hub
-RUN useradd -ms /bin/bash edgehubuser
+ARG EDGEHUBUSER_ID=1000
```

Comment:
Does this apply to x64 images?

Reply:
Same answer as before. Anca will take care of that part on her change.